src/app/base/components/export/export-helper.ts

```diff
@@ -24,7 +24,7 @@ export enum ExportPreset {
 
 export enum ExportRowType {
     ALL = "ALL",
-    SLICE = "SLICE",
+    SLICE = "DATA_SLICE",
```

Comment:
somewhat dangerous, we should change the enum entry as well (SLICE = "DATA_SLICE" to DATA_SLICE = "DATA_SLICE")